composer.json

```diff
@@ -20,6 +20,7 @@
     "require": {
         "php": ">= 7.1",
         "ext-json": "*",
+        "GuzzleHttp/guzzle": "^6.3",
```

Comment:
As guzzle are only used for a test, this should be in `require-dev`

Reply:
Thanks